### PR TITLE
add -o and -g options.

### PR DESCRIPTION
**Pull request by t-mangoe**
### Description

I added `-o` and `-g` options.

- Relevant Issues : #174 
- Relevant PRs : (none)
- Type of change :
  - [x] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [x] Addition or Improvement of tests
  - [ ] Addition or Improvement of documentation

### CODE REVIEW

**Comment by avdv** [comment]
Thanks for the PR! I am going to take a deeper look soon.

What I have seen so far: could you also add `-G` / `--no-group` option ("in a long listing, don't print group names") while at it?

**Comment by t-mangoe** [comment]
@avdv 
Thank you for your comment!
> could you also add -G / --no-group option ("in a long listing, don't print group names") while at it?

I got it. I'll try to challenge it.

**Comment by t-mangoe** [comment]
I added `-G` option.

File: spec/color_ls/flags_spec.rb [hunk 1]
```diff
@@ -325,4 +325,156 @@
...
+        :executable? => false
+      )
+
+      allow(ColorLS::FileInfo).to receive(:new).with("#{FIXTURES}/a.txt", link_info: true) { fileInfo }
+    end
+
+    it 'lists without group info' do
+      expect { subject }.not_to output(/sys/).to_stdout
+    end
+    it 'lists without user info' do
```
Comment (avdv) [inline spec/color_ls/flags_spec.rb:476]: The check is correct, but the description says the opposite.
```suggestion
    it 'lists with user info' do
```

**Review by avdv — APPROVED** [review]
Awesome work, thank you @t-mangoe! I only found a single glitch...

**Pull request merged**